CMakeLists.txt

```diff
@@ -168,14 +168,14 @@ endif()
 # catch2
 
 if(SYMFORCE_BUILD_BENCHMARKS OR SYMFORCE_BUILD_TESTS)
-  find_package(Catch2 3.0.0 QUIET)
+  find_package(Catch2 3.1.1 QUIET)
```

Comment:
I think we should leave this at 3.0.0, since symforce is compatible with any 3.x release (and you won't have a particular 3.x release installed and available if it doesn't build on your system).  Although the version checking semantics in cmake are somewhat fuzzy/nonspecified ([1](https://cmake.org/cmake/help/latest/command/find_package.html#find-package-version-format) [2](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-version-file)), I think this is basically the convention at least.  If it's 3.1.1 here, we might e.g. end up fetching and using our own copy of Catch2 on a system that already has 3.0.0 installed

Reply:
Makes sense. I changed it to just be 3.